templates/revoke.sh.j2

```diff
@@ -7,7 +7,7 @@ cd ${CADIR}
 
 gen_crl () {
   # regenerate the certificate revocation list
-  openssl ca -gencrl -config ca.conf -out ca-crl.pem
+  openssl ca -gencrl -config ca.conf -out ca-crl.pem -passin pass:$certkeypass
```

Comment:
Another password use - where is this coming from? Rolled your own CA outside of this playbook?